MessageHandler.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <binaryninjaapi.h>
+
+class MessageHandler {
+
+    std::set<uint64_t> m_msgSendFunctions;
+    static std::set<uint64_t> FindMsgSendFunctions(BinaryNinja::Ref<BinaryNinja::BinaryView> data);
+
+public:
+    MessageHandler(BinaryNinja::Ref<BinaryNinja::BinaryView> data);
+
+    std::set<uint64_t> GetMessageSendFunctions() const { return m_msgSendFunctions; }
+    bool HasMessageSendFunctions() const { return m_msgSendFunctions.size() != 0; }
+    bool IsMessageSend(uint64_t);
```

Comment:
These methods should use `camelCase` since that's what is used in the rest of the project. After #41, I am open to changing the convention across the board to `PascalCase` to match the core. However, I don't want to mix conventions in the meantime.